os/mm/mm_heap/mm_manage_allocfail.c

```diff
@@ -99,6 +105,31 @@ void mm_manage_alloc_fail(struct mm_heap_s *heap, int startidx, int endidx, size
 	abort_mode = true;
 #endif
 
+#ifdef CONFIG_DEBUG_MM_HEAPINFO
+	struct watchdog_status_s wd_status;
+	bool is_wd_paused = false;
+	mfdbg("Temporarily pause active watchdog\n");
+	int fd = open(CONFIG_WATCHDOG_DEVPATH, O_RDWR);
+	if (fd < 0) {
+		mfdbg("Fail to open %s, errno %d\n", CONFIG_WATCHDOG_DEVPATH, get_errno());
+	} else {
+		/* success open watchdog, try to get watchdog state */
+		if (ioctl(fd, WDIOC_GETSTATUS, (unsigned long)&wd_status) != OK) {
+			mfdbg("Fail to get watchdog state, errno %d\n", get_errno());
```

Comment:
It's minor, how about modifying the comment more clearly like this?
// If watchdog is successfully opened, try to get watchdog state.

Reply:
thanks!.. but 'watchdog is successfully opened' could  be wrong...., because It is watchdoc driver.
It can be understood as a watchdoc function.
so Could i change `if successfully open`.

Reply:
I understand. Then change to "if watchdog driver successfully opened,~"

Reply:
yes, Thank you for review!~ chaged